test/tests/fs/xhrFS/listing.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Unit tests for MountableFileSystem's mount/unmount features.
+ */
+var fs = require('fs'),
+    assert = require('wrapped-assert');
+
+module.exports = function() {
+  var oldRootFS = fs.getRootFS();
+
+  var rootForXFS = new BrowserFS.FileSystem.InMemory();
```

Comment:
Why create an in-memory FS? You should initialize the root to be the XHRFS you create w/ the listings object.